docs/platform_concepts/studio/train/entities.md

```diff
@@ -3,159 +3,188 @@ title : Entities
 sidebar_label : Entities
 ---
 
+An entity refers to specific information or data that the chatbot can identify and extract from user input. Let's consider the following example:
 
-In this article, you will learn: 
+User query: I'm looking for Italian restaurants in New York to accommodate four people.
 
-1. [Introduction](#ex)
-2. [How to add and train entities](#ate)
-3. [Types of entities](#et)
-4. [Autoskipping prompts using entities](#ape)
+In this sentence, the entities are the location (New York), cuisine (Italian), and the number of people (four).
 
-:::tip
-You can access Entities using the keyboard shortcut `e`.
-:::
+By training your bot on these specific entities, it enables the bot to answer questions like this accurately. Entities are primarily utilized to bypass prompts and provide suggestions to users once the bot identifies the relevant entity. For more information on configuring entities for this purpose, please refer to this section.
```

Comment:
Please refer is not required

Reply:
Thanks for pointing it out, I have to hyperlink a section there.